registry/server/templates/routes/updateTemplate.ts

```diff
@@ -24,32 +25,30 @@ const validateRequestBeforeUpdateTemplate = validateRequestFactory([
 ]);
 
 const updateTemplate = async (req: Request<UpdateTemplateRequestParams>, res: Response): Promise<void> => {
-    const template = {
-        content: req.body.content,
-    };
-    const templateName = req.params.name;
-
-    const templatesToUpdate = await db('templates').where({
-        name: templateName,
-    });
-    if (!templatesToUpdate.length) {
-        res.status(404).send('Not found');
-        return;
-    }
-
-    const localizedVersions = req.body.localizedVersions || {};
-    const localesAreValid = await validateLocalesAreSupported(Object.keys(localizedVersions), res);
-    if (!localesAreValid) {
-        return;
+    const result = await templatesRepository.updateTemplate(
+        req.params.name,
+        req.body,
+        // TODO: cheeck if user here is really required
```

Comment:
I'd better to handle it right away rather than leaving `TODO` 
Let us know what piece of info is missing to handle it